Tests/Fixtures/MatthiasDependencyInjectionTestExtension.php

```diff
@@ -5,11 +5,20 @@
 use Symfony\Component\DependencyInjection\Definition;
 use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 use Symfony\Component\Config\FileLocator;
 
-class MatthiasDependencyInjectionTestExtension implements ExtensionInterface
+class MatthiasDependencyInjectionTestExtension implements ExtensionInterface, PrependExtensionInterface
```

Comment:
I really don't think that there is any actual benefit from it -> especially if invocation of prepend method is conditioned with load method.